### ISSUE TITLE
Any way to skip the initial Select+Start calibration?

### ISSUE THREAD

**Issue by RaulDJ** [cboulay/PSMoveService#458]
Hi,

I'm using RiftCat with my smartphone and a lightened up ping-pong ball on top of my HMD headband for tracking and my problem is that before doing the calibration, the PS Move controller postion and orientation is almost perfect: it's not perfectly offsetted to my HMD but both 3D spaces (PS Move and HMD spaces) match, and I can move and rotate over myself without seeing the controller flying around.

But when I do the Select+Start calibration, it just **destroys** everything: PS Move orientation get's f*cked up no matter how perfectly aligned and stable you hold it and the position, well, it's just completely missaligned with the HMD as soon as you start moving or rotating.

I can easily hide the "need_alignment_gesture.png" message but I cannot use the controller outside the Steam overlay menu unless I do the calibration.

Any help with this?

Thank you very much.

**Comment by HipsterSloth** [comment]
Sorry about that. The alignment setup can use some work. There is currently no way to disable the alignment gesture completely, though that would be easy to add (I can add that in the next release). However you can disable the orientation portion of the alignment by adding the following to the "psmove_settings portion of your steamvr.vrsettings file:

```
"psmove_settings":{
    "use_orientation_in_alignment":false
}
```

**Comment by RaulDJ** [comment]
Yes, I read about that setting on another issue but sadly the worst offender is the position offset. Either way I think I'll be stopping with all this Android based VR as my phone (Moto G 4º generation) seems to have some extremely shitty accelerometers and no compass whatsoever, so drifting is insane and I have to recalibrate the orientation every 30 seconds or so, making everything unplayable. I might get something like a Oculus DK2 sometime in the future to at least have proper orientation tracking.

Thank you very much to you and to cboulay for this software though, you're doing a great work with it.
